spec/exhibits-solr-conf/configure_solr_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'rake'
+require 'hurley'
+require 'jettywrapper'
+
+describe 'exhibits namespace rake task' do
+  before do
+    load File.expand_path('../../../lib/exhibits-solr-conf/tasks/configure_solr.rake', __FILE__)
+  end
+
+  describe 'configure_solr' do
+    it 'takes argument for the target solr conf dir' do
+      target_dir = "#{File.dirname(__FILE__)}/../../tmp/solr/"
+      FileUtils.remove_dir(target_dir) if Dir.exist?(target_dir)
+      Dir.mkdir(target_dir)
+      Rake::Task['exhibits:configure_solr'].reenable
+      Rake.application.invoke_task "exhibits:configure_solr[#{target_dir}]"
+      copied_files = Dir.entries(target_dir)
+      expect(copied_files.size).to be > 5
+      expect(copied_files).to include('schema.xml', 'solrconfig.xml', 'synonyms_both_anchors.txt')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.